modules/kernel/src/main/scala-2.10/notebook/ReplCalculator.scala

```diff
@@ -170,6 +171,7 @@ class ReplCalculator(
     implicit val ec = context.dispatcher
 
     private var queue: Queue[(ActorRef, ExecuteRequest)] = Queue.empty
+    private var currentlyExecutingTask: Option[Future[(String, EvaluationResult)]] = None
```

Comment:
later, when we need we could add `sender actor ref`, `executeRequest` or some more data...